src/recon_test/test_blocks_on_cylindrical_projectors.cxx

```diff
@@ -905,6 +989,8 @@ BlocksTests::run_tests()
   print_time("map orientation test took: ");
   run_intersection_with_cylinder_test();
   print_time("intersection with cylinder test took: ");
+//  run_back_projection_test_with_axial_buckets(back_projector);
+//  print_time("back projection test with axial buckets took: ");
```

Comment:
Re-enable this test but use only 1 axial bucket. A future PR to allow for multiple axial buckets.